hep-eppsu-software.tex

```diff
@@ -228,9 +227,9 @@ \subsection{Hyper-K and DUNE}\label{hyper-k-and-dune}
 overcome. The field is heavily reliant upon nuclear research, which is often
 funded for other applications. Technical barriers already solved for the LHC
 still remain an issue for the neutrino community due to a lack of dedicated
-effort. E.g., there is no common event format (although one has 
+effort. e.g., there is no common event format (although one has 
```

Comment:
This isn't correct - at the start of a sentence it should be "E.g.". However, it's actually better to change the preceding full stop to a comma.